### PR TITLE
Improve focus of project owner definition

### PR DESCRIPTION
**Pull request by Kriesse**
Also improve spelling consistency

### CODE REVIEW

**Review by yoshuawuyts — APPROVED** [review]
:sparkles:

**Comment by yoshuawuyts** [comment]
LGTM

**Comment by okdistribute** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ Each project should be assigned a person, or 'owner,' who is responsible for tha
...
+The 'project owner' is responsible for that project being released. This person keeps the issues clean, up-to-date and relevant to the repository, merges major pull requests, and schedules releases. This role keeps the team focused on a particular goal. 
 
-## Making decisions
+## Making Decisions
 
-Although we have 'Project Owners,' there is a sense of contribution to all the projects by all of the team members and the surrounding open source ecosystem. Any decisions made in regards to the project are probably fine as long as we are all considering the opinions of other folks on the team, are nice people generally, and willing to make compromises.
+While we have 'Project Owners,' there is a sense of contribution to all the projects by all of the team members and the surrounding open source ecosystem. Any decisions made in regards to the project ar fine as long as we are all considering the opinions of other folks on the team, are nice people generally, and willing to make compromises.
```
Comment (ralphtheninja) [inline README.md]: ar -> are

**Pull request merged**